Resources/Prototypes/Catalog/Fills/Lockers/service.yml

```diff
@@ -130,7 +130,7 @@
 - type: entity
   id: ClosetJanitorBombFilled
   parent: ClosetJanitorBomb
-  suffix: Filled
+  suffix: DO NOT MAP, Filled
```

Comment:
why dont map?

Reply:
parent has DNM suffix, that one entity also is filled